tools/remote_build/linux.bazelrc

```diff
@@ -54,6 +54,10 @@ build:msan --test_timeout=60,900,1800,3600
 build:msan --test_tag_filters=-no_linux,-nomsan,-json_run_localhost
 build:msan --cxxopt=--stdlib=libc++
 build:msan --linkopt=--stdlib=libc++
+# use MSAN-instrumented version of libc++
+# TODO(jtattermusch): include the modified LD_LIBRARY_PATH in an msan-specific
+# toolchain configuration, instead of needing to explicitly set it here.
+build:msan --action_env=LD_LIBRARY_PATH=/usr/local/libcxx-msan/lib/x86_64-unknown-linux-gnu
```

Comment:
qq: until now, we haven't been setting any special flavor of libc++ for TSAN. Since the rbe_ubuntu2004 image now has also the TSAN-instrumented version of libstdc++ installed, it might be a good idea to use it. I'm leaving that as a possible followup (I don't want to change more stuff than necessary in this PR).